cf/s3-v2.yaml

```diff
@@ -0,0 +1,265 @@
+AWSTemplateFormatVersion: "2010-09-09"
+Description: Set up S3 buckets for Cellenics [managed by github.com/hms-dbmi-cellenics/iac]
+
+Parameters:
+  Environment:
+    Type: String
+    Default: development
+    AllowedValues:
+      - development
+      - staging
+      - production
+    Description: The environment for which the buckets need to be created.
+
+Resources:
+  PlotsTablesBucket:
+    Type: AWS::S3::Bucket
+    Properties:
+      BucketName: !Sub "plots-tables-${Environment}-${AWS::AccountId}"
+      PublicAccessBlockConfiguration:
+        BlockPublicAcls: True
+        BlockPublicPolicy: True
+        IgnorePublicAcls: True
+        RestrictPublicBuckets: True
+      BucketEncryption:
+        ServerSideEncryptionConfiguration:
+          - ServerSideEncryptionByDefault:
+              SSEAlgorithm: 'aws:kms'
+
+  CellSetsBucket:
+    Type: AWS::S3::Bucket
+    Properties:
+      BucketName: !Sub "cell-sets-${Environment}-${AWS::AccountId}"
```

Comment:
Names are changed, encryption added, other than that the file is a copy of s3.yaml